### PR TITLE
Added methods skipAction, tryToMakeAction to ToolMakerHutFields, changed newTurn,  updated adaptors, typo in GameBoard

### LINKED ISSUE #31: Make newTurn imply end of game

### PR DESCRIPTION
**Pull request by rvadovic**
Closes #31

### CODE REVIEW

File: src/main/java/sk/uniba/fmph/dcs/game_board/ToolMakerHutFields.java [hunk 1]
```diff
@@ -70,6 +70,17 @@ public boolean canPlaceOnToolMaker(final Player player) {
...
 
+    public boolean tryToMakeActionToolMaker(final Player player) {
+        return toolMakerFigures.contains(player.playerOrder());
+    }
+
+    public boolean skipActionToolMaker(final Player player) {
+        if (tryToMakeActionToolMaker(player)) {
+            toolMakerFigures.clear();
+        }
+        return true;
```
Comment (Fremal31) [inline src/main/java/sk/uniba/fmph/dcs/game_board/ToolMakerHutFields.java:81]:
```suggestion
            toolMakerFigures.clear();
            return true;
        }
        return false;
```

Ja to mam ze vrati false ak tam nema figurky, ale asi je to jedno lebo aj tak to ma na starosti GamephaseController. Takto je to mozno aj lepsie aby skip sa vzdy podaril lebo skoro nic nerobi

File: src/main/java/sk/uniba/fmph/dcs/game_board/ToolMakerHutFields.java [hunk 2]
```diff
@@ -124,8 +145,22 @@ public boolean canPlaceOnFields(final Player player) {
...
 
+    public boolean tryToMakeActionFields(final Player player) {
+        return fieldsFigures.contains(player.playerOrder());
+    }
+
+    public boolean skipActionFields(final Player player) {
+        if (tryToMakeActionToolMaker(player)) {
+            fieldsFigures.remove(player.playerOrder());
+        }
+        return true;
```
Comment (Fremal31) [inline src/main/java/sk/uniba/fmph/dcs/game_board/ToolMakerHutFields.java:156]: To iste ako vyssie

**Review by Fremal31 — COMMENTED** [review]
Vyzera ok 👍. Zatial nedam approve aby ostatny mohli nazbierat PR reviews

File: src/main/java/sk/uniba/fmph/dcs/game_board/ToolMakerHutFields.java (hunk 1 above)
Comment (rvadovic) [reply src/main/java/sk/uniba/fmph/dcs/game_board/ToolMakerHutFields.java:81]: Tak to necham tak ako to mam

**Review by MatusBucek — APPROVED** [review]
OK

**Pull request merged**